openreview/conference/templates/SUPERpaperBidWebfield.js

```diff
@@ -0,0 +1,518 @@
+
+  var loadingContent = Handlebars.templates.spinner({ extraClasses: 'spinner-inline' });
+  sections = [
+    {
+      heading: 'All Papers  <span class="glyphicon glyphicon-search"></span>',
+      id: 'allPapers',
+      content: null
+    }
+  ];
+
+  BID_OPTIONS.forEach(function(option) {
+    sections.push({
+      heading: option,
+      headingCount: bidsById[option].length,
+      id: option.replace(' ', '').toLowerCase(),
+      content: loadingContent
+    })
+  });
+
+  sections[activeTab].active = true;
+
+  $('#notes .tabs-container').remove();
+
+  Webfield.ui.tabPanel(sections, {
+    container: '#notes',
+    hidden: true
+  });
+
+  // Render the contents of the All Papers tab
+  var searchResultsOptions = _.assign({}, paperDisplayOptions, { container: '#allPapers' });
+  var submissionListOptions = {
+    heading: null,
+    container: '#allPapers',
+    search: {
+      enabled: true,
+      localSearch: false,
+      subjectAreas: SUBJECT_AREAS,
+      subjectAreaDropdown: 'basic',
+      invitation: BLIND_SUBMISSION_ID,
+      sort: false,
+      onResults: function(searchResults) {
+        Webfield.ui.searchResults(prepareNotes(searchResults, conflictIds, bidsByNote), searchResultsOptions);
+      },
+      onReset: function() {
+        Webfield.ui.searchResults(notes, searchResultsOptions);
+        $('#allPapers').append(view.paginationLinks(noteCount, 50, 1));
+      },
+    },
+    displayOptions: paperDisplayOptions,
+    autoLoad: false,
+    noteCount: noteCount,
+    pageSize: 50,
+    onPageClick: function(offset) {
+      var sortedNotesP;
+      if (_.startsWith(selectedScore, 'SUPER_')) {
+        sortedNotesP = $.when(getAffinityScores(), getBidsCount())
+          .then(function(affinityScores, bidsCount) {
+            return getPapersSortedByAlgo(getPaperScores(affinityScores, bidsCount), offset);
+          });
+      } else {
+        sortedNotesP = getPapersSortedByAffinity(offset);
+      }
+      return sortedNotesP
+      .then(function(notes) {
+        return prepareNotes(notes, conflictIds, bidsByNote);
+      });
+    },
+    fadeIn: false
+  };
+
+  Webfield.ui.submissionList(notes, submissionListOptions);
+
+
+  if (SCORE_IDS.length) {
+
+    var optionsHtml = '';
+    SCORE_IDS.forEach(function(scoreId) {
+      var label = view.prettyInvitationId(scoreId);
+      var selectedClass = selectedScore == scoreId && 'selected';
+      optionsHtml = optionsHtml + '<option value="' + scoreId + '" ' + selectedClass + '>' + label + '</option>';
+    })
+
+    if (ENABLE_SUPER_ALGO) {
+      SCORE_IDS.forEach(function(scoreId) {
+        var label = 'SUPER ' + view.prettyInvitationId(scoreId);
+        var superScoreId = 'SUPER_' + scoreId;
+        var selectedClass = selectedScore == superScoreId && 'selected';
+        optionsHtml = optionsHtml + '<option value="' + superScoreId + '" ' + selectedClass + '>' + label + '</option>';
+      })
+    }
```

Comment:
Yes, organizers can have different scores such as OpenReview affinity scores, TPMS scores or Semantic Scholar scores and so on. All of these scores could use your algorithm if it's enabled and decide to sort it like that.
Maybe an option to ONLY use your algorithm with any of the scores should also be there.

Reply:
Which of the those scoring methods are between 0 and 1?

Reply:
That's a good question. I know that OpenReview affinity scores are 0 to 1 and I think TPMS is also 0 to 1, but that's not necessarily always the case. I'm not sure if normalizing them would work for your algorithm. What do you think?

Reply:
I think normalizing them would probably be okay. Maybe for now though it would make the most sense to restrict to the openreview affinity scores? What do you think?

Reply:
Yes, that works for me